src/components/FilterPanel.jsx

```diff
@@ -9,77 +9,221 @@ import {
+        />
+      ),
+      count: params.grades?.length ?? 0,
+      wasChanged:
+        JSON.stringify(grades?.length > 0 ? grades : undefined) !==
+        JSON.stringify(params.grades),
+    },
   };
 
+  function resetFilters() {
+    setMinAmount(undefined);
+    setMaxAmount(undefined);
+    setGrades(undefined);
+    setMajors(undefined);
+  }
+
+  function cancelChanges() {
+    setMinAmount(params.minAmount);
+    setMaxAmount(params.maxAmount);
+    setGrades(params.grades);
+    setMajors(params.majors);
+  }
+
+  const filtersChanged =
+    filters.Major.wasChanged ||
+    filters.Amount.wasChanged ||
+    filters['Grade Level'].wasChanged;
+
+  const filterCount =
+    filters.Major.count + filters.Amount.count + filters['Grade Level'].count;
+
+  useEffect(() => {
+    setFilterCount(filterCount);
+  }, [filterCount, setFilterCount]);
+
   return (
     <Box>
       <Toolbar
         disableGutters
-        sx={{ display: { md: 'none' }, justifyContent: 'space-between' }}>
-        <IconButton onClick={onClose}>
+        sx={{
+          justifyContent: 'space-between',
+          alignContent: 'flex-end',
+        }}>
+        <IconButton onClick={onClose} sx={{ visibility: { md: 'hidden' } }}>
           <CloseIcon />
         </IconButton>
 
         <Typography>Filters</Typography>
 
-        <Button disabled>Reset</Button>
+        <Button
+          disabled={
+            !params.minAmount &&
+            !params.maxAmount &&
+            (!params.grades || params.grades?.length === 0) &&
+            (!params.majors || params.majors?.length === 0)
+          }
+          onClick={resetFilters}>
+          Reset
+        </Button>
       </Toolbar>
 
+      {filterCount > 0 && (
+        <AccordionFilter name="Active Filters" defaultExpanded>
+          {params.majors?.map((e) => (
+            <FilterChip label={e} />
+          ))}
+          {Number.isInteger(params.minAmount) && (
+            <FilterChip label={`Min $${params.minAmount}`} />
+          )}
+          {Number.isInteger(params.maxAmount) && (
+            <FilterChip label={`Max $${params.maxAmount}`} />
+          )}
+          {params.grades?.map((e) => (
+            <FilterChip label={GradeLevel.toString(e)} />
+          ))}
+        </AccordionFilter>
+      )}
+
       {Object.entries(filters).map(([name, filter]) => (
-        <Accordion key={name} disableGutters>
-          <AccordionSummary
-            expandIcon={<ExpandMoreIcon />}
-            aria-controls={name + '-content'}
-            id={name + '-header'}>
-            <Typography>{name}</Typography>
-          </AccordionSummary>
-
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
-        </Accordion>
+        <AccordionFilter name={name} count={filter.count}>
+          {filter.comp}
+        </AccordionFilter>
       ))}
+
+      {filtersChanged && (
+        <Alert severity="warning">Your changes have not yet been applied</Alert>
```

Comment:
Didn't remove border since it looks weird when the filter panel spans the entire screen width. 

<img width="683" alt="Screen Shot 2022-02-09 at 4 46 04 PM" src="https://user-images.githubusercontent.com/40483569/153315302-e1c1e327-afee-45d8-bec2-c0b3c35321e6.png">
VS
<img width="682" alt="Screen Shot 2022-02-09 at 4 47 02 PM" src="https://user-images.githubusercontent.com/40483569/153315396-a51c81d4-38a8-40c9-a009-4fbe1a555f5a.png">